scripts/portable-sbob.py

```
@@ -0,0 +1,151 @@
+def fix_endpoint(e):
+    """Volatile Host headers -> the single-label DNS wildcard."""
+    changed = []
+    hosts = ((e.get("headers") or {}).get("Host")) or []
+    new = []
+    for h in hosts:
+        host = h.split(":")[0]
+        port = h.split(":")[1] if ":" in h else None
+        try:
+            ipaddress.ip_address(host)
+        except ValueError:
+            new.append(h); continue
```

Ruff (0.16.0)

[error] Multiple statements on one line (semicolon)

(E702)